conftest.py

```diff
@@ -27,11 +25,12 @@ def pytest_configure(config):
 
 
 def pytest_unconfigure(config):
-    db.test.drop_tables(models)
-    db.test.close()
+    db.database.drop_tables(models)
```

Comment:
We always need the prefix for tests, and once the tests finished, there is no risk that the same DB instance will be used for production in the same thread ;)